docs/git/cheat-sheet.md

```diff
@@ -1,9 +1,55 @@
-# Git Cheat Sheet (Detailed)
+# Git Cheat Sheet
 
-See also
-[fixing commits][fix-commit]
+See also [fixing commits][fix-commit]
 
-Commands:
+## Common Commands
+
+- add
+    - `git add foo.py`
+- checkout
+    - `git checkout -b new-branch-name`
+    - `git checkout master`
+    - `git checkout old-branch-name`
+- commit
+    - `git commit --amend`
+- config
+    - `git config --global core.editor nano`
+    - `git config --global core.symlinks true`
+- diff
+    - `git diff`
+    - `git diff --cached`
+    - `git diff HEAD~2..`
+- fetch
+    - `git fetch origin`
+    - `git fetch upstream`
+- grep
+    - `git grep update_unread_counts -- '*.js'`
+- log
+    - `git log`
+- pull
+    - **do not use for Zulip**
```

Comment:
Maybe add `git pull --rebase` and document why normal pull is undesirable in Zulip.